sriov/tests/SR_IOV_BondVF_DPDK/README.md

```diff
@@ -0,0 +1,50 @@
+## Test Case Name: SR-IOV.BondVF.DPDK
+
+### Objective(s): Test and ensure that VFs DPDK bond (mode 0, 1) across PFs work as expected.
```

Comment:
Can we outline clearly what these bond modes are, and what is expected in every case?

Reply:
We shouldn't use test spec as education doc; these are industry standard and well known